React-WhatsNewInReact18.MD

```diff
@@ -5,7 +5,7 @@
 Let's take a look at the major new features of React 18: 
 
 ## Root API (new feature)
-React 18 introduces Root API `ReactDOM.createRoot`. Before React 18, we used `ReactDOM.render` to render a component to the page. Going forward with React 18, we will use ReactDOM.createRoot to create a root, and then pass the root to the render function. The good news is that your current code with `ReactDOM.render` will still work, however, it is recommended to start transitioning to `createRoot` as `render` will be marked `deprecated` starting React 18.
+React 18 introduces a new Root API: `ReactDOM.createRoot`. Before React 18, we used `ReactDOM.render` to render a component to the page. Going forward with React 18, we will use `ReactDOM.createRoot` to create a root, and then pass the component to the render function. The good news is that your current code with `ReactDOM.render` will still work, however, it is recommended to start transitioning to `createRoot` as `render` will be marked `deprecated` starting React 18.
```

Comment:
first you say "`ReactDOM.render` to render a component" then  "pass the root to the render function" and below in the demo you added the comment "//render app to root". The three different wordings might be confusing. So I suggested making it consistent with "pass component to render" across the board.